docs/scos/dev/guidelines/keeping-a-project-upgradable/running-the-evaluator-tool.md

```diff
@@ -20,61 +20,36 @@ This document describes how to check if code is compliant with Spryker’s stand
 
 ## Prerequisites
 
-To start working with the evaluator tool, connect to the Docker SDK CLI container:
-
-```bash
-docker/sdk cli
-```
-
 Get general information about the tool and see all available commands related to the evaluation process we may in *analyze* section:
```

Comment:
```suggestion
Get general information about the tool and see all available commands related to evaluation in the `analyze` section:
```

Reply:
What are the other sections for?

Reply:
We use SDK as a general tool, where the evaluator is only one of the other sniffers